networkx/algorithms/tests/test_tournament.py

```diff
@@ -92,6 +125,32 @@ def test_hamiltonian_cycle(self):
         assert path[0] in G[path[-1]]
 
 
+class TestScoreSequence:
+    """Unit tests for the :func:`networkx.tournament.score_sequence`
+    function.
+
+    """
+
+    def test_score_sequence_edge(self):
+        G = DiGraph([(0, 1)])
+        assert score_sequence(G) == [0, 1]
+
+    def test_score_sequence_triangle(self):
+        G = DiGraph([(0, 1), (1, 2), (2, 0)])
+        assert score_sequence(G) == [1, 1, 1]
+
+
+class TestTournamentMatrix:
+    """Unit tests for the :func:`networkx.tournament.tournament_matrix`
+    function.
+
+    """
+
+    def test_tournament_matrix(self):
+        G = DiGraph([(0, 1)])
+        assert tournament_matrix(G) == sparse.coo_matrix([[0, 1], [-1, 0]])
```

Comment:
This is a tricky test for multiple reasons:
 1. It fails in it's current form due to the value comparison amongst arrays (see CI failure)
 2. It's actually a bit confusing because the output types aren't consistent: `tournament_matrix` returns a `csr_matrix`, not `coo`.

It's not clear whether the type component of the test was intentional, but from context I assume it's not.

If the point of the test is merely to assert that all of the values are correct, then I'd do something like:

```suggestion
        m = tournament_matrix(G)
        npt.assert_array_equal(m.todense(), np.array([[0, 1], [-1, 0]])
```

where `npt` is an alias for `numpy.testing` e.g. `npt = pytest.importorskip("numpy.testing")`.

If you also wanted to test the output type, you could add an `isintance(m, sp.sparse.csr.csr_matrix)`, but I don't think that's necessary.